### ISSUE TITLE
Add ability to squish snap-content rather than pushing it off the page

### ISSUE THREAD

**Issue by robertsoniv** [jtrussell/angular-snap.js#116]
First of all, I love this module- it works so well!

Something that may make this more usable on desktop apps is an option for controlling how the `snap-content` get's moved out of the way for the incoming `snap-drawers`.  Desktop applications will often allow users to keep the drawer open while continuing to work within the main content view.

Right now, if the left drawer is open, the `snap-content` will be partially off of the right side of the page. I've been able to work around this using CSS and media queries. Instead of actually opening the snapper, on desktop screen sizes I'm adding `left:@drawer-width` to the `snap-content`.

Basically if there were a setting where you could choose to either **_push_** the snap-content out of the way (the way it works now), **_overlay_** the drawer over the `snap-content`, or **_squish_** the `snap-content` so that both the drawer and the content are fully visible (the content would just resize to fit the remaining space).

That way, I could choose to "squish" on desktop and "push" on mobile and use one snapper & the snapper methods for activating/deactivating the menu.

Thanks for all of your previous work and in advance for any responses/suggestions you might have!
-Rob

**Comment by jtrussell** [comment]
Hi @robertsoniv - this is definitely an interesting feature idea. To be honest though it's probably better addressed in the snap.js project itself. We're more or less providing an Angular friendly wrapper and aren't really equipped to accommodate this sort of thing :).

**Comment by robertsoniv** [comment]
Good point! I didn't even think of that
